File: app/src/main/java/com/fomchenkovoutlook/artem/android_web_browser_example/WebBrowserView.java
package com.fomchenkovoutlook.artem.android_web_browser_example;

import android.annotation.SuppressLint;
import android.content.Context;
import android.support.annotation.NonNull;
import android.util.AttributeSet;
import android.webkit.WebView;

// View:
public class WebBrowserView extends WebView {

    public static final String HTTP_PROTOCOL = "http://";
    public static final String HTTPS_PROTOCOL = "https://";

    public WebBrowserView(Context context) {
        super(context);
    }

    public WebBrowserView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public WebBrowserView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @SuppressLint("ClickableViewAccessibility")
    @Override
    public boolean performClick() {
        return true;
    }
}

File: app/src/main/java/com/fomchenkovoutlook/artem/android_web_browser_example/WebBrowserView.kt
package com.fomchenkovoutlook.artem.android_web_browser_example

import android.annotation.SuppressLint
import android.content.Context
import android.util.AttributeSet
import android.webkit.WebView

/**
 * Clickable WebView
 */
class WebBrowserView: WebView {

    constructor(context: Context?) : super(context)
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)

    @SuppressLint("ClickableViewAccessibility")
    override fun performClick(): Boolean {
        return true
    }

}